RecoLocalCalo/HcalRecProducers/plugins/alpaka/HBHERecHitProducerPortable.cc

```diff
@@ -0,0 +1,169 @@
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "CondFormats/DataRecord/interface/HcalSiPMCharacteristicsRcd.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalSiPMCharacteristicsDevice.h"
+#include "CondFormats/DataRecord/interface/HcalMahiConditionsRcd.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalMahiConditionsDevice.h"
+#include "CondFormats/DataRecord/interface/HcalRecoParamsRcd.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalRecoParamWithPulseShapeDevice.h"
+#include "CondFormats/HcalObjects/interface/alpaka/HcalMahiPulseOffsetsDevice.h"
+
+#include "DataFormats/HcalDigi/interface/alpaka/HcalDigiDeviceCollection.h"
+#include "DataFormats/HcalRecHit/interface/alpaka/HcalRecHitDeviceCollection.h"
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/Event.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/EventSetup.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/EDGetToken.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/EDPutToken.h"
+#include "HeterogeneousCore/AlpakaCore/interface/alpaka/stream/EDProducer.h"
+#include "HeterogeneousCore/CUDACore/interface/JobConfigurationGPURecord.h"
+
+#include "Mahi.h"
+#include "CalibCalorimetry/HcalAlgos/interface/HcalTimeSlew.h"
+
+namespace ALPAKA_ACCELERATOR_NAMESPACE {
+
+  class HBHERecHitProducerPortable : public stream::EDProducer<> {
+  public:
+    explicit HBHERecHitProducerPortable(edm::ParameterSet const&);
+    ~HBHERecHitProducerPortable() override;
```

Comment:
```suggestion
    ~HBHERecHitProducerPortable() override = default;
```
and remove the explicit empty body below.

Reply:
done in 6ba6f2400bad40a6f2c057fda1e488559181d93a